testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/OIDCAdvancedRequestParamsTest.java

```diff
@@ -307,6 +306,7 @@ public void promptNoneSuccess() {
 
     // Prompt=none with consent required for client
     @Test
+    @DisableFeature(value = Profile.Feature.ACCOUNT2, skipRestart = true) // TODO remove this (KEYCLOAK-16228)
```

Comment:
Skipped this tests, as it causes to fail other 5 tests, as it is unable to revert consentRequired to false at the end of the test.